examples/anime-http/server/asyncapi.yaml

```diff
@@ -39,11 +39,12 @@ operations:
       $ref: '#/channels/trendingAnime'
     messages:
       - $ref: '#/components/messages/trendingAnime'
-    reply:
-      channel:
-        $ref: "#/channels/trendingAnime"
-      messages:
-        - $ref: '#/components/messages/trendingAnimeReply'
+  trindingAnimeReply:
```

Comment:
This is a reply of the previous operation. Why putting it here as a separate one? 🤔

Reply:
@fmvilas, based on the spec, a reply can be sent to any channel of any server without providing a way for the user to add bindings to the reply field.
Glee can't operate without bindings for a lot of protocols like HTTP. For example, if you can't tell it which method it should use to send a message to an HTTP server, sending the message becomes impossible.

that's why we have decided to not support the `request/reply` pattern for now till we have the bindings in the reply.
https://github.com/asyncapi/spec/issues/1009

why remove the reply in Operation Functions?
since spec also has a reply field and it has a different meaning:
*in spec*: reply to any known or unknown channel on any server.
*in glee*: sending a response back to the same channel and server.

since the difference of meanings can confuse the user, we have decided to not support the reply for version 1.0 of glee for now till https://github.com/asyncapi/spec/issues/1009 is resolved and we can have complete support of the request/reply pattern.

Reply:
Ok, that makes sense. Thanks for the explanation 🙏